modules/draco/src/draco-encoder.js

```diff
@@ -74,10 +81,13 @@ export default class DRACOEncoder {
 
     try {
       const encodedLen =
-        this.dracoEncoder.EncodePointCloudToDracoBuffer(dracoPointCloud, dracoData);
+        this.dracoEncoder.EncodePointCloudToDracoBuffer(dracoPointCloud, false, dracoData);
```

Comment:
This is where it was broken - missing a second argument.

Reply:
(Makes me wonder what effect changing that middle boolean parameter would have).